.gitignore

```diff
@@ -9,5 +9,4 @@
 /dist
 /node_modules
 /yarn-error.log
-/gemfiles/*.lock
```

Comment:
I don't know why we ignored the lockfiles, but enough time has passed that missing lockfiles do now break the build because the versions we'll install without lockfiles are so wildly different that nothing works anymore.

Instead, I created the **ONE** Lockfile we need with

```
BUNDLE_GEMFILE=gemfiles/Gemfile-rails-7-0 bundle lock
BUNDLE_GEMFILE=gemfiles/Gemfile-rails-7-0 bundle lock --add-platform x86_64-linux
```